### ISSUE TITLE
unable to load resolver "node"

### ISSUE THREAD

**Issue by renanborgez** [RyoNkmr/eslint-plugin-atomic-design#141]
Hello @RyoNkmr 

I'm getting this strange error when using your plugin, do you have any idea about what is it?
![image](https://user-images.githubusercontent.com/1863024/95459267-b5bec700-0973-11eb-8a5b-96ee81d3dc7a.png)

I've tried using the default configuration, with custom folders, and disabling everything, and it still appearing when the plugin is enabled.

my versions:

```
Node: 12.18.2 (also tested with version 10 and 14 and the problem persists)
Npm: 6.14.8
eslint-plugin-atomic-design: 0.2.2
eslint: 6.5.1
babel-eslint: 10.1.0 (I also tested without babel-eslint but the problem persists)
```

**Comment by RyoNkmr** [comment]
I'm sorry I know v0.2.2 is stuck with outdated ESLint and Node:

https://github.com/RyoNkmr/eslint-plugin-atomic-design/blob/8ca1c27aef75dd5e94b11a77d1c81eb8dd40b821/package.json#L51-L56

please try now I released 1.0 with too many dependencies updates.

**Comment by renanborgez** [comment]
Hello RyoNkmr
Many thanks for your hard work.
I tried to install the version v1.0.0 but I'm getting the following error:

![image](https://user-images.githubusercontent.com/1863024/112025893-f3846600-8b35-11eb-94c0-6ac312d1b7a5.png)

Altought the version v0.2.2 installs fine (but of course still that error as showed in the issue)
my node version: v14.15.4 and npm is: 7.5.6
